mappings/net/minecraft/recipe/RecipeSerializer.mapping

```diff
@@ -1,10 +1,10 @@
-CLASS azo net/minecraft/recipe/RecipeSerializers
+CLASS azo net/minecraft/recipe/RecipeSerializer
```

Comment:
I've done a few changes already, make sure to take them into account.

Reply:
Ok. I renamed `RecipeSerializers` -> `RecipeSerializer` as it's an interface all serializers implement, not just the place where serialziers are stored.